#!/usr/bin/env python
PACKAGE = "pips"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("param_name", str_t, 0, "robot_description")
gen.add("floor_tolerance", double_t, 0, "Floor tolerance",    .03, 0,   1)
gen.add("safety_expansion", double_t, 0, "Safety expansion",    .02, 0,   1)

exit(gen.generate(PACKAGE, "pips", "RobotModel"))
